src/components/clerkcustom/invite-form.tsx

```diff
@@ -0,0 +1,207 @@
+        setLoading(true)
+
+        const formData = new FormData(e.currentTarget)
+        const firstName = formData.get('firstName')?.toString() ?? ''
+        const lastName = formData.get('lastName')?.toString() ?? ''
+        const email = formData.get('email')?.toString() ?? ''
+        const password = formData.get('password')?.toString() ?? ''
+
+        // Check if organization is selected
+        if (!selectedOrganization) {
+            setError('Please select an organization')
+            setLoading(false)
+            return
+        }
+
+        // Check if at least one role is selected
+        if (!formValues.roles.reviewer && !formValues.roles.researcher) {
+            setError('Please select at least one role (Reviewer or Researcher)')
+            setLoading(false)
+            return
+        }
+
+        // Update form values state
+        setFormValues({
+            firstName,
+            lastName,
+            email,
+            password,
+            roles: formValues.roles, // Keep the existing roles structure
+        })
+
+        try {
+            // save user in Clerk
+            const result = await createClerkUserAction({
+                firstName,
+                lastName,
+                email,
+                password,
+            })
+            // save user in DB
+            await createUserAction(
+                result.clerkId,
+                `${firstName} ${lastName}`,
+                formValues.roles.researcher,
+                selectedOrganization, // Use the selected organization ID directly
+                formValues.roles.reviewer,
+            )
+
+            if (result.success) {
+                // Reset form state
+                setFormValues({
+                    firstName: '',
+                    lastName: '',
+                    email: '',
+                    password: '',
+                    roles: {
+                        reviewer: false,
+                        researcher: false,
+                    },
+                })
+                setSuccess(true)
+            }
+        } catch (error: unknown) {
+            const errorMessage = error instanceof Error ? error.message : 'User creation failed'
+            setError(errorMessage)
+        } finally {
+            setLoading(false)
+        }
+    }
+
+    return (
+        <form onSubmit={handleSubmit}>
+            {error && (
+                <Alert icon={<Warning size={16} />} color="red" mb="md">
+                    {error}
+                </Alert>
+            )}
+
+            {success && (
+                <Alert color="green" mb="md">
+                    User created successfully!
+                </Alert>
+            )}
+            <div
+                style={{
+                    display: 'flex',
+                    width: '100%',
+                    gap: '1rem',
+                    alignItems: 'flex-end',
+                    marginBottom: '1rem',
+                }}
```

Comment:
two suggestions: 
* Mantine has a lot of components to do this sort of thing, such as Group, Stack, Flex, each of which have props for setting styles such as `mb` (for marginBottom) etc.
* I'd prefer to use raw style calls only as a last resort, instead we can use the style from pandacss if we really do need to apply custom styling